components/MonsterAbilities.vue

```diff
@@ -72,7 +72,7 @@
 
 <script lang="ts" setup>
 import { useFormatModifier } from '@/composables/useFormatModifier';
-import { useDiceRoller } from '@/composables/useDicerRoller';
```

Comment:
Just a typo here. It was "Dicer Roller" and I assumed that was unintentional, so updated it to "Dice Roller"